l10n_it_corrispettivi_sale/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,9 @@
+**Italiano**
+
+Sull'ordine di vendita, è possibile impostare il flag *Corrispettivi*, ereditato dal partner.
+Quando un ordine di vendita ha il flag *Corrispettivi* abilitato, la fattura da esso generata è un corrispettivo.
+
+**English**
+
+In the sale order, you can set the *Corrispettivi* flag, inherited from the partner.
+When a sale order has the *Corrispettivi* flag, the invoice generated by it is a corrispettivo.
```

Comment:
Per "flag" vedi i suggerimenti indicati in #868.


Corrispettivi → Sums due
anche in altre parti del readme e dove compare nelle stringhe sorgente del codice.

Reply:
> Per "flag" vedi i suggerimenti indicati in #868.
> 
> Corrispettivi → Sums due
> anche in altre parti del readme e dove compare nelle stringhe sorgente del codice.

In base a quanto visto in 
https://github.com/OCA/l10n-italy/pull/977#discussion_r255326522

Corrispettivi → Ricevute/Receipts